mixer/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM scratch
+
+# obtained from debian ca-certs deb using fetch_cacerts.sh
+ADD ca-certificates.tgz /
+# All containers need a /tmp directory
+WORKDIR /tmp/
```

Comment:
What generates files into `/tmp/`?

Reply:
It's generally faster to mount a memory volume than writing to union FS in docker

Reply:
I think we should consistently use the same workdir in istio. 

Let's get back to this on a separate PR - I'm not entirely sure where sysstemd sets the workdir,
and how to harmonize this with the raw vms.

Reply:
If it is only for the logs - I would use /var/log for the output. 

But I'm not entirely sure if it's healthy for a k8s job to fill up tmp or /var/log - normally logs 
go to stderr and are captured/rotated/etc. How much disk do we get by default on the pod ?